### PR TITLE
Use generated verstr.h in DDMD to include the generated git tag

### PR DESCRIPTION
**Pull request by yebblies**
@WalterBright

### CODE REVIEW

**Comment by WalterBright** [comment]
Auto-merge toggled on

File: src/magicport/dprinter.d [hunk 1]
```diff
@@ -153,7 +153,7 @@ class DPrinter : Visitor
         switch(s)
         {
         case "NULL": print("null"); return;
-        case "__IMPORT__": print("\"v\" ~ import(\"VERSION\")[0 .. $ - 1]"); return;
+        case "__IMPORT__": print("\"v\" ~ import(\"verstr.h\")[1 .. 6]"); return;
```
Comment (MartinNowak) [inline src/magicport/dprinter.d]: It's something like `2.069-devel-23b065f-dirty` on posix.mak.

Comment (MartinNowak) [inline src/magicport/dprinter.d]: https://github.com/D-Programming-Language/dmd/pull/4903

Comment (yebblies) [inline src/magicport/dprinter.d]: Oops, I don't know how I forgot about that.

**Pull request merged**